### ISSUE TITLE
Hi, interesting project!

### ISSUE THREAD

**Issue by baruchiro** [ryanrodemoyer/AzureDevOps_GraphQL#2]
I didn't develop a GraphQL server before. From reading your code, I guess we need to build a schema for all the Azure DevOps REST APIs?

**Comment by ryanrodemoyer** [comment]
> guess we need to build a schema for all the Azure DevOps REST APIs

maybe? I suppose it depends on what you're trying to build. In this case of this project, this is a facade layer over the Azure DevOps REST api. I wanted to create a way to make API calls to Azure DevOps using GraphQL - but I still have to use their REST api under the hood. I had to create GraphQL types for the stuff I want to expose from ADO.

**Comment by baruchiro** [comment]
Yes, I think it's my point. If for example I want to query Git REST endpoints, I need to create types and mapping from graphql to the ADO
